account_avatax/models/account_move.py

```diff
@@ -13,10 +13,45 @@ class AccountMove(models.Model):
 
     @api.depends("partner_shipping_id", "partner_id", "company_id")
     def _compute_onchange_exemption(self):
-        for move in self:
-            if not move.exemption_locked and not hasattr(move, "partner_shipping_id"):
-                move.exemption_code = move.partner_id.property_exemption_number
-                move.exemption_code_id = move.partner_id.property_exemption_code_id.id
+        """
+        Set the exemption to use for the Invoice.
+        An exemption can be applied if
+        there an exemption for the delivery address Country + State
+        - Get the delivery address Country & State
+        - Search the invoiced commercial partner addresses
+          for an exemption in this Country & State
+        - In case there is a "country wide" exemption, use it.
+
+        Example:
+        - ACME company partner, USA CA, has exemption status
+        - ACME Invoicing address, USA CA, no exemption status
+        - ACME Delivery address, USA CA, no exemption status
+
+        Invoice to ACME Invoicing, Shipped to ACME Delivery with be exempt.
+
+        For this to work properly, the "exemption_lock" is no longer supported.
+        """
+        for invoice in self:
+            invoice_partner = invoice.partner_id.commercial_partner_id
+            ship_to_address = invoice.shipping_add_id
```

Comment:
There is no shipping_add_id in Odoo base code, it is partner_shipping_id

Reply:
Fixed, and build is green now